### PR TITLE
Adds the Covid Live App to the list of sites that are using svelte

### PR DESCRIPTION
**Pull request by krmichael**
I am rewriting the Covid Live App Dashboard at svelte <3, could you add the dashboard logo to the list?

Thanks!!

### CODE REVIEW

**Comment by swyxio** [comment]
Deploy Preview is done!

Built with commit 375a7046dd393747055b005a5e80fb2d1574bbf4

https://deploy-preview-420--svelte-community.netlify.app

File: whos-using-svelte/WhosUsingSvelte.js [hunk 1]
```diff
@@ -1,4 +1,10 @@
 export const companies = [
+	{
+		href: "https://covidlive.app/",
+		src: "assets/covidlive-logo.jpeg",
+		alt: "Covid Live App logo",
+		prominent: true,
```
Comment (swyxio) [inline whos-using-svelte/WhosUsingSvelte.js:6]:
```suggestion
```

File: whos-using-svelte/WhosUsingSvelte.js [hunk 2]
```diff
@@ -1,4 +1,10 @@
 export const companies = [
+	{
+		href: "https://covidlive.app/",
+		src: "assets/covidlive-logo.jpeg",
```
Comment (swyxio) [inline whos-using-svelte/WhosUsingSvelte.js:4]: asset missing

Comment (LeanderG) [reply whos-using-svelte/WhosUsingSvelte.js:4]: image is not showing up on https://svelte.dev/

**Comment by frederikhors** [comment]
I think they are using React right now, not Svelte.

We should remove it.

**Pull request merged**